pkg/network/infraconfigurators/masquerade.go

```diff
@@ -171,17 +142,12 @@ func (b *MasqueradePodNetworkConfigurator) PreparePodNetworkInterface() error {
 }
 
 func (b *MasqueradePodNetworkConfigurator) GenerateDomainIfaceSpec() api.Interface {
-	domainIface := api.Interface{
-		MTU: &api.MTU{Size: strconv.Itoa(b.podNicLink.Attrs().MTU)},
-		Target: &api.InterfaceTarget{
-			Device:  b.handler.GenerateTapDeviceName(b.podNicLink.Attrs().Name),
-			Managed: "no",
-		},
-	}
-	if b.vmMac != nil {
-		domainIface.MAC = &api.MAC{MAC: b.vmMac.String()}
-	}
-	return domainIface
+	// The method is left here since currently the DomainIfaceSpec cache is used as a marker that phase1 was completed
+	return api.Interface{}
+}
+
+func (b *MasqueradePodNetworkConfigurator) GenerateDHCPConfig() *cache.DHCPConfig {
+	return nil
```

Comment:
Ok, this answers partially my concerns above (how it can succeed compiling). 

But I cannot understand nor reason about why you're doing this. Never the less, please move this to where the method was previously so reviewers can easily see the diffs.

Reply:
It's on the wrong commit, I will move it to the correct one,